src/components/Vocal.js

```diff
@@ -77,6 +82,14 @@ const Vocal = ({
 		startRecognition()
 	}
 
+	const _onFocus = () => {
+		buttonRef.current.style.outline = '2px solid'
```

Comment:
If this is used for styling information only, it should have the same display conditions as other styling instructions (ie className === null)

Reply:
Right